platform/ui/src/contextProviders/ContextMenuProvider.js

```diff
@@ -0,0 +1,136 @@
+import React, {
+  createContext,
+  useContext,
+  useEffect,
+  useCallback,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import ToolContextMenu from '../../../viewer/src/connectedComponents/ToolContextMenu';
```

Comment:
This import is across package boundaries?

Reply:
Done. Components are now passed through props. thanks.